recipes/pycurl/pycurl.inc

```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "PycURL is a Python interface to libcurl."
+LICENSE = "LGPL-2.1"
+
+inherit python-module
+
+DEPENDS = "libcurl curl-dev native:util/curl-config libssl"
+SRC_URI = "https://dl.bintray.com/pycurl/pycurl/pycurl-${PV}.tar.gz"
+SRC_URI += "file://0001-disable-autodetection-of-libcurl-linkerflags.patch"
+S = "${SRCDIR}/pycurl-${PV}"
+
+EXTRA_PYCONF = " \
+    --with-openssl \
```

Comment:
meh, we also have libre! :) but that is ok to leave for those needed it if you don't care